### ISSUE TITLE
Package System.Security.Principal.Windows 5.0.0 is not compatible with net6.0

### ISSUE THREAD

**Issue by jfradkin** [oqtane/oqtane.framework#1773]
Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1202	Package System.Security.Principal.Windows 5.0.0 is not compatible with net6.0 (.NETCoreApp,Version=v6.0). Package System.Security.Principal.Windows 5.0.0 supports: uap10.0.16299 (UAP,Version=v10.0.16299)	Oqtane.Server	C:\S\S\DEV\oqtane\oqtane.framework\Oqtane.Server\Oqtane.Server.csproj	1

**Comment by jfradkin** [comment]
I cloned the repo per instructions that builds, not sure why I get this error from the downloaded version.
If any one else runs into it clone using github desktop.

**Comment by jfradkin** [comment]
was too quick I could swear I did a build and it built no errors but when I went to run I get same build error.
Since its not a package included in nuget im not even sure how to fix. I guess I can go back to the master release that ran for me, but honestly I think im done. you guys have fun.

**Comment by sbwalker** [comment]
Are you able to describe the steps to reproduce? It sounds like you were trying to run the last Oqtane release (2.3.1) on .NET 6? And then you cloned the code from the dev branch (3.0.0) over top of the 2.3.1 version and tried to build?

**Comment by jfradkin** [comment]
sorry im not very with it, but what i did was go from the main page to clone the repo.
https://github.com/oqtane/oqtane.framework
first I tried the download zip. second I tried the clone option using github desktop.
Both of these projects I am using vs 2022 and I could swear I did a build successfully on the cloned version, but when I try to run I get the error. I was able to build and run the master which is .net 5, but I would like to use the hot deploy in vs 2022 and I found that works with .net 6.
At this point I have deleted it all as I couldn't get it working except in .net 5 
I may pull down the master again and experiment, but im afraid its all over my head :)

**Comment by sbwalker** [comment]
The code in the main branch has not been upgraded to .NET 6 - it is the code for the Oqtane 2.3.1 release which is based on .NET 5. The code in the dev branch has been upgraded to .NET 6 and will be the basis of the Oqtane 3.0.0 release which will be published this week. If you want to run on .NET 6 and experiment with Hot Reload then I would suggest you clone the dev branch. My personal experience with Hot Reload has not been productive - it slows down the debugging experience considerably in VS2022 and I actually disabled it in my IDE. I am sure it will get better over time as this is essentially the V1 release.

**Comment by leigh-pointer** [comment]
@jfradkin you need to clone the dev branch master is the release branch oqtane 2.3.1 .net5! 

On another point hot refresh doesn’t work to good at the moment.

**Comment by jfradkin** [comment]
I guess I dont know how to clone the dev release, sorry for not being up to speed.

**Comment by jfradkin** [comment]
I can just use master for the time being it seems to be a very rich environment. I did my last project using mud blazor but unless I dont understand I can still use that with modules, although I saw they added a bunch to blazor with vs 2022

**Comment by jfradkin** [comment]
I will google how to clone a dev release or branch etc. I should know how. thanks for all the help.
I disagree on hot reload I was very happy to have it (although a bit late) as I work with doing a UI its faster than stopping the project at least it seemed that way to me on my last project. which is done, but I did convert to .net 6. I had to re-create it as it was giving me odd errors just changing the .net
